packages/website/assets/routes/index.js

```diff
@@ -1,171 +1,196 @@
 
+// This is an autogenerated file. See the routes package for more information.
 export default [
   {
-    hash: 'pcc-cx-social',
-    title: 'CX Social',
-    description: 'Intended for gravel and CX bikes!',
-    distance: 62,
-    vert: 758,
-    type: 'off-road',
-    link: 'https://ridewithgps.com/routes/34684204',
-    gpx: 'https://ridewithgps.com/routes/34684204.gpx',
-    fit: 'https://ridewithgps.com/routes/34684204.fit',
-    map: '/assets/routes/pcc-cx-social.png'
+    "hash": "pcc-geoff-s-suburban-london-social",
```

Comment:
@achingbrain dunno why it didn't jsonify before? or did you prev just run `standard --fix` after or something? either way i chucked it in the linter ignore cos its autogenerated - shout if you disagree